### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by Zia123456**
- Data updated
- Data queried
vet_clinic_5
vet_clinic_6
vet_clinic_7
vet_clinic_8
vet_clinic_9

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -11,3 +11,24 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
```
Comment (HAMMAS-SALEEM) [inline data.sql:11]: ## 1
image
## 2
image
## 3
image
## 4
image

- [x] Your readme file is not really good-looking. Kindly remove the sections which I have shown in the images above arrow_up: to make it descriptive and professional :+1:

Comment (HAMMAS-SALEEM) [inline data.sql:11]: ## 1. Put your project name in the desctiption
image

## 2. Put your comment in this section
image

- [x] Kindly add the information in these sections which are shown in the image above :+1: to make your readme descriptive :blush:

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

## Hey @Zia123456 :wave:

## Highlights :1st_place_medal:
:heavy_check_mark: Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 2]
```diff
@@ -1,10 +1,48 @@
...
+/* Tansaction */
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+/* Tansaction */
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2020-01-01';
```
Comment (Lucas-Masaba) [inline queries.sql]: - [x] The date here should have been "Jan 1st, 2022" but instead you have "Jan 1st, 2020". Please fix that 👍

File: data.sql [hunk 3]
```diff
@@ -11,3 +11,24 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
...
+VALUES ('Charmander', '2020-02-08', 11, false, 0);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Plantmon', '2021-11-15', 5.7, true, 2);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Squirtle', '1993-04-02', 12.13, false, 3);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Angemon', '2005-06-12', 45, true, 1);
```
Comment (Lucas-Masaba) [inline data.sql]: - [x] According to the project requirements, all these pokemon weights should be negative values.
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Zia123456  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Most requirements were met 👍 
- Descriptive pull request 📜 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Zia123456, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [x] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "bug fixed".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**